app/frontend/components/pages/staking/stakingPage.js

```diff
@@ -0,0 +1,33 @@
+const {h} = require('preact')
+
+const StakingPage = () =>
+  h(
+    'div',
+    {class: 'staking-wrapper'},
+    h('div', {class: 'staking-label'}, 'Upcoming'),
```

Comment:
The icon seems slightly off vertical center

Reply:
Fixed, but check on the retina, please.